tools/mksurfdata_map/src/mksurfdat.F90

```diff
@@ -745,21 +745,23 @@ program mksurfdat
           pftdata_mask(n) = 1
        end if
 
-       ! Make sure sum of land cover types does not exceed 100. If it does,
-       ! subtract excess from most dominant land cover.
-       
-       suma = pctlak(n) + pctwet(n) + pcturb(n) + pctgla(n)
+       ! Make sure sum of all land cover types except natural vegetation 
+       ! does not exceed 100. If it does, subtract excess from these land cover
+       ! types proportionally.
+
+       suma = pctlak(n) + pctwet(n) + pcturb(n) + pctgla(n) + pctcft(n)%get_pct_l2g()
        if (suma > 250._r4) then
```

Comment:
Removed (two places).  The rest of this check (if (suma > 100._r4) then...) will eventually be moved into normalizencheck_landuse as in #1579 .  I'm unsure whether to do that here now, but I'm inclined to leave it as is and do it as part of a manual merge of this PR with #1579 . Let me know if you feel otherwise.

Reply:
Sounds like a good plan - thanks.